tests/Feature/MemberRegistrationTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Tests\Feature;
+
+use Tests\TestCase;
+
+class MemberRegistrationTest extends TestCase
+{
+    private $fakeMemberData = [
+        'voornaam' => 'Berend',
+        'tussenvoegsel' => 'van',
+        'achternaam' => 'Zuid',
+        'geboortedatum' => '1980-10-10',
+        'geslacht' => 'M',
+        'adres' => 'Boegmeen 27',
+        'postcode' => '1234 AB',
+        'plaats' => 'Lalastad',
+        'telefoon' => '0123456789',
+        'email' => 'berend@vanzuid.nl',
+        'selected_camp' => '1',
+        'eindexamen' => 'VWO',
+        'studie' => 'Nietsnuttigheid',
+        'afgestudeerd' => 1,
+        'vak1' => 1,
+        'klas1' => 1,
+        'hoebij' => ['je moeder', 'je vader'],
+        'vog' => 1,
+        'privacy' => 1
+    ];
+    
+    private function clearDB(): void
+    {
+        \App\Member::where('email', $this->fakeMemberData['email'])->delete();
```

Comment:
Zelfde als  mijn eerdere comment, check `use DatabaseTransactions` als trait op tests . Enige waar je voor moet zorgen is dat je storage engine op `InnoDb` staat. dit kan je doen in de `config/database.php` , je kan mijn `feature/generic-comments` checken daar heb ik het werkend gekregen.